components/inputs/test/input-number.visual-diff.js

```diff
@@ -0,0 +1,95 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
```

Comment:
For the visual diff tests, I think we want to be careful that we're not creating a lot of tests that just do the same thing as the unit tests.

Unit tests are super fast and really easy to write, and are great for things like checking that the validation code is resulting in the errors we expect, or that setting a particular value is resulting in the formatted value we expect, events are firing when they should, etc.

Visual tests are slow, but are good at making sure a particular state is rendered properly. So I'd suggest only doing visual tests for things like: default, hidden label, required, disabled, placeholder, invalid (no focus), invalid (with focus), invalid (focus then blur -- if that was a bug you had trouble with). No need to have a visual test for every single type of validation error, or every single type of number output, since those are easy to write unit tests for.